app/src/main/res/values-de/strings.xml

```diff
@@ -2,22 +2,22 @@
     <string name="error_invalid_domain">Ungültige Domain angegeben</string>
     <string name="error_failed_app_registration">Diese App konnte sich auf dem Server nicht authentifizieren.</string>
     <string name="error_authorization_unknown">Ein undefinierbarer Autorisierungsfehler ist aufgetreten.</string>
-    <string name="error_compose_character_limit">Der Status ist zu lang!</string>
+    <string name="error_compose_character_limit">Der Beitrag ist zu lang!</string>
     <string name="error_media_upload_size">Die Datei muss kleiner als 4MB sein.</string>
     <string name="error_media_upload_type">Dieser Dateityp darf nicht hochgeladen werden.</string>
     <string name="error_media_upload_opening">Die Datei konnte nicht geöffnet werden.</string>
     <string name="error_media_upload_permission">Eine Leseberechtigung wird für das Hochladen der Mediendatei benötigt.</string>
-    <string name="error_media_upload_image_or_video">Bilder und Videos können beide nicht an den Status angehängt werden.</string>
+    <string name="error_media_upload_image_or_video">Bilder und Videos können beide nicht an den Beitrag angehängt werden.</string>
     <string name="error_media_upload_sending">Die Mediendatei konnte nicht hochgeladen werden.</string>
-    <string name="error_report_too_few_statuses">Mindestens ein Status muss berichtet werden.</string>
+    <string name="error_report_too_few_statuses">Mindestens ein Beitrag muss berichtet werden.</string>
 
     <string name="title_home">Start</string>
     <string name="title_notifications">Benachrichtigungen</string>
     <string name="title_thread">Unterhaltung</string>
     <string name="title_tag">#%s</string>
-    <string name="title_statuses">Posts</string>
-    <string name="title_follows">Follows</string>
-    <string name="title_followers">Follower</string>
+    <string name="title_statuses">Beiträge</string>
+    <string name="title_follows">Folgt</string>
```

Comment:
Nah, "Folgungen" sounds a bit weird and "Ich folge" is only for the user. Would be funny if you're on someone's profile and then it reads "Ich folge" instead of "Folgt".